packages/forms-web-app/__tests__/unit/validators/full-planning/granted-or-refused.test.js

```diff
@@ -0,0 +1,81 @@
+const { validationResult } = require('express-validator');
+const {
+  rules,
+  validApplicationDecisionOptions,
+} = require('../../../../src/validators/full-planning/granted-or-refused');
+const { testExpressValidatorMiddleware } = require('../validation-middleware-helper');
+
+describe('validators/full-planning/granted-or-refused', () => {
+  describe('rules', () => {
+    it('is configured with the expected rules', () => {
+      expect(rules().length).toEqual(1);
+    });
+  });
+
+  describe('ruleGrantedOrRefused', () => {
+    it('is configured with the expected rules', () => {
+      const rule = rules()[0].builder.build();
+
+      expect(rule.fields).toEqual(['granted-or-refused']);
+      expect(rule.locations).toEqual(['body']);
+      expect(rule.optional).toBeFalsy();
+      expect(rule.stack).toHaveLength(3);
+
+      expect(rule.stack[0].negated).toBeTruthy();
+      expect(rule.stack[0].validator.name).toEqual('isEmpty');
+      expect(rule.stack[0].message).toEqual(
+        'Select if your planning application was granted or refused, or if you have not received a decision'
+      );
+
```

Comment:
Not sure if intentional but it seems the an entry has been skipped within the stack array.

Reply:
Stack[1] is nothing but bail. So its intentional.